release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -129,6 +146,21 @@ the {product-title} web console.
 The `KubeletTooManyPods` alert now uses the Pod capacity metric as a threshold
 instead of a fixed number.
 
+[id="ocp-4-3-monitor-your-own-services"]
+==== Monitor your own Services (Technology Preview)
```

Comment:
Based on our current glossary usage for _service_, I wonder if this needs to be capitalized? (_Capitalize when referring to the Kubernetes object specifically._) Are we talking about some particular application service or specifically the Service API object? I'm guessing the former.

(https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#service)

Reply:
Yeah, this isn't specifically referring to the Service API object. I'll lowercase this instance of *service*.